router/api/sockets.js

```diff
@@ -25,11 +25,24 @@ module.exports = function(app) {
           if (err) {
             console.log("Could not join session");
             io.emit("error", err);
-          } else {
-            socket.join(data.sessionId);
-            console.log("Session joined:", session._id);
-            io.emit("sessions", SessionCtrl.getSocketSessions());
-            io.to(session._id).emit("session-change", session);
+          }
+          else {
+            const numOfMembersInSession =
+              io.nsps['/'].adapter.rooms[data.sessionId] === undefined ?
+              1 :
+              io.nsps['/'].adapter.rooms[data.sessionId].length;
+
+            if (numOfMembersInSession <= 2) {
```

Comment:
This should be strictly less than, right?

```suggestion
            if (numOfMembersInSession < 2) {
```